python/GafferArnold/__init__.py

```diff
@@ -49,15 +49,20 @@
 
 	import sys
 	import ctypes
-	originalDLOpenFlags = sys.getdlopenflags()
-	sys.setdlopenflags( originalDLOpenFlags & ~ctypes.RTLD_GLOBAL )
+	import os
+
+	if os.name != "nt":
```

Comment:
> I'm wondering in this case if it might be more accurate to use hasattr( ctypes, "RTLD_GLOBAL" )?

That does seem better and more descriptive of why it's included or excluded.

> Is there any difference between testing os.name and sys.platform as you've done in the other file?

I should have used `sys.platform` since it's already being imported, they're functionally interchangeable, aside from the need to test for different values. But assuming `hasattr()` works, I think that will be even better.

Reply:
Fixed in bbb93e3ebc58ac683da2a8a7591347018106cc6d

Reply:
Looks like `python/IECoreArnold/__init__.py` and `startup/GafferScene/usd.py` are still using different version of the test...

Reply:
Ack! Good catch. Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/4750/commits/043f68e7bb394526ca3957fb15874b0d16cd320b